k9mail/src/main/java/com/fsck/k9/service/NotificationActionService.java

```diff
@@ -237,10 +312,21 @@ public int startService(Intent intent, int startId) {
             } else if (ACKNOWLEDGE_ACTION.equals(action)) {
                 // nothing to do here, we just want to cancel the notification so the list
                 // of unseen messages is reset
+                Log.i(K9.LOG_TAG, "notification acknowledged");
             }
 
-            /* there's no point in keeping the notification after the user clicked on it */
-            controller.notifyAccountCancel(this, account);
+            // if this was a stacked notification on Android Wear, update the summary
+            // notification and keep the other stacked notifications
```

Comment:
This comment states that the summary notification is updated. Where does that happen?